tools/docker/bin/uninstall.sh

```diff
@@ -16,15 +16,14 @@ truncate -s 0 /var/www/html/wp-content/debug.log
 # Ensure wp-config.php doesn't have multi-site settings
 echo
 echo "Clearing out possible multi-site related settings from wp-config.php"
-echo "It's okay to see errors if these did't exist..."
-wp --allow-root config delete WP_ALLOW_MULTISITE
-wp --allow-root config delete MULTISITE
-wp --allow-root config delete SUBDOMAIN_INSTALL
-wp --allow-root config delete base
-wp --allow-root config delete DOMAIN_CURRENT_SITE
-wp --allow-root config delete PATH_CURRENT_SITE
-wp --allow-root config delete SITE_ID_CURRENT_SITE
-wp --allow-root config delete BLOG_ID_CURRENT_SITE
+wp --allow-root config delete WP_ALLOW_McccULTISITE 2>&1 | grep -v "not defined" || true
```

Comment:
err?
```suggestion
wp --allow-root config delete WP_ALLOW_MULTISITE 2>&1 | grep -v "not defined" || true
```

Reply:
Ugh. Thanks for catching that. My system started acting up and I guess keyboard input did go through into the wrong window. 

I like your fancy though. I'll look at that before requesting a new review.